server/svix-server/Cargo.toml

```diff
@@ -54,6 +54,7 @@ time = { version = "0.3.9", features = [ "std" ]}
 futures = "0.3"
```

Comment:
Also: remove the wait-for from the docker file (both its usage, and copying it over). Remove it from the repo too.
Make sure though that we wait for the same time as before in the docker file.